### PR TITLE
Add PostgreSQL db

### PR DESCRIPTION
**Pull request by michalboruch**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,9 +1,17 @@
 version: '2'
 services:
+  db:
+    image: postgres
```
Comment (janeek1995) [inline docker-compose.yaml]: should not we have specific version of postgres here ? I guess it means :leatest

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,9 +1,17 @@
 version: '2'
 services:
+  db:
```
Comment (janeek1995) [inline docker-compose.yaml]: we should add volumes here.

File: docker-compose.yaml (hunk 1 above)
Comment (michalboruch) [reply docker-compose.yaml]: done

File: docker-compose.yaml (hunk 2 above)
Comment (michalboruch) [reply docker-compose.yaml]: done

**Pull request merged**